sdk/cdn/Azure.ResourceManager.Cdn/api/Azure.ResourceManager.Cdn.net8.0.cs

```diff
@@ -2051,6 +2050,16 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Cdn.Models.DeliveryRuleAction>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Cdn.Models.DeliveryRuleAction>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    public abstract partial class DeliveryRuleActionParameters : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Cdn.Models.DeliveryRuleActionParameters>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.Cdn.Models.DeliveryRuleActionParameters>
```

Comment:
We should avoid a type to end with `Parameters`.
Please add a `rename-mapping` entry to rename this to `DeliveryRuleActionProperties`, like:
```
rename-mapping:
  DeliveryRuleActionParameters: DeliveryRuleActionProperties
```
and then regenerate the code and export the api.